wiki/FAQ/en.md

```diff
@@ -238,121 +446,215 @@ Key | Bindings
+It can be found here: <https://osu.ppy.sh/forum/t/131067>
 
-Also, [the ranked beatmap version of osu! tutorial][osu!tutorial_rank] is good for practicing some of the possible mapping design you will come across. It's an excellent training map for newbies and experienced players.
+For _osu!mania_, **Entozer** has compiled a decent list sorted by relative player's difficulties, with **Drace** providing an experienced gameplay insight for beginners starting out in _osu!mania_.
+Both of it can be found here: <https://osu.ppy.sh/forum/t/147552> (**Entozer**'s beatmap guide), and <https://osu.ppy.sh/forum/t/146615> (**Drace**'s improvement guide)
 
-Alternatively, you can [sort the ranked beatmap list by difficulty][sort beatmaplist difficulty]. The easiest maps are at the top of the list.
+Also, [the ranked beatmap version of osu! tutorial][osu!tutorial rank] is good for practicing some of the possible mapping design.
+It is an excellent training map for newbies and experienced players.
+
+Alternatively, the website's beatmap searchbar has a feature to [sort the ranked beatmap list by difficulty][sort beatmaplist difficulty].
+The easiest maps are at the top of the list.
 
 #### What does Level in my profile actually do?
-Absolutely _nothing_. There is no difference in any way when comparing a Level 100 player against a Level 1 player except the player's own skills and experience of play. There is no handicap for the lower level nor there is an advantage for the higher level (except personal experience and hardware used).
+**Absolutely nothing**.
+There is no difference in any way when comparing a Level 100 player against a Level 1 player except the player's own skills and experience of the game mode in question.
+There is no handicap for the lower level nor is there an advantage for the higher level (except personal experience and hardware used).
 
-Level is based on your total score accumulated when online with your account whether you pass or fail the beatmap. [Formula given here][Score#Level wikilink].
+Level will be based on total score accumulated when online with account regardless of pass or fail on the beatmap.
+[Formula given here][Score#Level wikilink].
 
 #### Why is my profile stats different ingame?
-Keep in mind that osu! has four game modes so you might be looking at the wrong game mode.
-
-Manually select the game mode at your profile that you want to view.
+Keep in mind that _osu!_ has four game modes so you might be looking at the wrong game mode.
 
 ![Profile_Mode_Buttons][Profile_Mode_Buttons image]
 
+_Manually select the game mode to view at the profile._
+
+#### How do I change the default keyboard key input for my game mode of choice?
+![Options sidebar with quickicons][Options quickicons image]
+
+From the _Input_ icon in the _Options_ sidebar as shown above, under **Keyboard** section as shown below.
+
+![Options keyboard section][Options keyboard image]
+
+#### Game over? Where did I messed up?
```

Comment:
Hmm, @tsudoko's suggestion seems ok, just replace `messed` with `mess`... unless you are suggesting for better wording of that heading?